docs/reference/rest-api/usage.asciidoc

```diff
@@ -254,7 +254,10 @@ GET /_xpack/usage
         "total" : 0,
         "failed" : 0
      }
-   }
+   },
+    "functions": {
+      ...
```

Comment:
nit: Maybe we should at least show how 1 or 2 functions are being reported here?

Reply:
I'll give it a try, it's still not completely clear to me how these values depend on the order of test executions.
If we pre-register all the functions this should be easier.

Reply:
I wasn't aware this is doctested. I didn't mean to increase the scope here, so let's only include examples here if it's little to no effort.

Reply:
Actually I'm still trying to make it pass... apparently the `...` is not enough to skip the section